webapp/channels/src/sass/components/_videos.scss

```diff
@@ -2,72 +2,132 @@
 @use "utils/variables";
 
 .post {
+    // Container for both video and thumbnail
     .video-div {
         position: relative;
-        overflow: hidden;
+        overflow: hidden; // Ensure hover effects stay contained
+        width: 100%;
         max-width: 480px;
-        height: 360px;
-        border-radius: 4px;
         margin-bottom: 8px;
 
-        iframe,
-        video {
+        iframe {
+            position: absolute;
+            top: 0;
+            left: 0;
             display: block;
-            max-width: 100%;
+            overflow: hidden;
+            width: 100%;
+            height: 100%;
+            border: none;
+            border-radius: var(--radius-s);
+            aspect-ratio: 16/9; // Always use 16:9
+            background-color: #000;
+        }
+
+        // Container for thumbnails
+        .video-thumbnail__container {
+            position: relative;
+            overflow: hidden;
+            width: 100%;
+            border-radius: var(--radius-s);
+            aspect-ratio: 16/9; // Always use 16:9
+            background-color: #000;
+            outline: none; // Remove default focus outline
+
+            // Focus state for keyboard navigation
+            &:focus-visible {
+                outline: 2px solid var(--button-bg);
+                outline-offset: 2px;
+            }
+
+            // Darkening overlay on hover
+            &::after {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 100%;
+                height: 100%;
+                background-color: rgba(0, 0, 0, 0);
+                content: '';
+                pointer-events: none;
+                transition: background-color 0.2s ease;
+            }
+
+            &:hover,
+            &:focus-visible {
+                &::after {
+                    background-color: rgba(0, 0, 0, 0.16);
+                }
+
+                .play-button {
+                    background-color: #FF0033; //youtube red
+                    transform: scale(1.02); // Reduced scale to prevent overflow
```

Comment:
I think this comment is incorrect since we're increasing the scale and it's nothing to do with overflow

Reply:
yep, removed